lib/brexit_checker/action.rb

```diff
@@ -29,4 +33,17 @@ def self.load_all
     @load_all = nil if Rails.env.development?
     @load_all ||= YAML.load_file(CONFIG_PATH)["actions"].map { |a| new(a) }
   end
+
+private
+
+  def load_groups
+    all_groups ||= YAML.load_file(GROUPS_PATH)["groups"]
```

Comment:
memoizing this does not work because it all_groups is a local variable

Reply:
ah good point, humm didn't like the idea of loading these each time, could I use `@@` here?
Looks like we should have a chat about this whole flow